bash/01_dataloading.sh

```diff
@@ -5,6 +5,5 @@ import_public dcp_pluto &
 import_public dcp_colp &
 import_public dcas_ipis &
 import_public dof_air_rights_lots &
-psql $BUILD_ENGINE -f sql/load_modifications.sql  
```

Comment:
We were debating about this as well. I believe we landed on this because it made development easier with need to rerun the modifcations tables every time and we think of modification as its own module therefore the loading and processing were moved closer together.